### ISSUE TITLE
JetEngine: When there's a lot of meta fields in custom post / options, the editing becomes slower and slower

### ISSUE THREAD

**Issue by Sapper-Morton** [CrocoBlock/suggestions#6457]
Creating a lot of meta fields slows down the process. Typing, changing values and adding new fields becomes really slow.

**To Reproduce**
1. Create 40+ meta fields
2. Notice slow down when editing said metafields. It might matter if you're editing metafield on the top or bottom.

**Expected behavior**
Editing should be fast

**Desktop (please complete the following information):**
 - OS: OSX Ventura 13.1 (M1 Mac)
 - Firefox
 - Version 109.0

**Additional context**
Basically this seems not to be a bug but a feature that causes the slowdown. JetEngine keeps refreshing everything on every keystroke and button press. Solution is to remove always on refreshing when keys are pressed and update stuff only when user switches fields.

**Comment by morosmo** [comment]
+1 same here
